clippy_lints/src/methods.rs

```diff
@@ -487,11 +513,13 @@ impl LintPass for Pass {
                     FILTER_NEXT,
                     FILTER_MAP,
                     ITER_NTH,
-                    ITER_SKIP_NEXT)
+                    ITER_SKIP_NEXT,
+                    GET_UNWRAP)
     }
 }
 
 impl LateLintPass for Pass {
+    #[allow(cyclomatic_complexity)]
```

Comment:
Please use the cyclomatic compexity level attribute to increase the threshold instead of simply allowing the lint